### ISSUE TITLE
Release 1.1.0 not available on pypi

### ISSUE THREAD

**Issue by blaze33** [jazzband/django-configurations#158]
I don't know if there's some delay on pypi's part or if a manual intervention is required but `pip install` does not work:

```bash
$pip install django-configurations==1.1.0
Collecting django-configurations==1.1.0
  Could not find a version that satisfies the requirement django-configurations==1.1.0 (from versions: 0.1, 0.2, 0.2.1, 0.3, 0.3.1, 0.3.2, 0.4, 0.5, 0.5.1, 0.6, 0.7, 0.8, 1.0, 2.0.dev0)
No matching distribution found for django-configurations==1.1.0
```

**Comment by codingjoe** [comment]
Apparently the [auto deploy](https://travis-ci.org/jazzband/django-configurations/jobs/137569622) failed.

```shell
Uploading distributions to https://pypi.python.org/pypi
Uploading django_configurations-2.0.dev0-py2.py3-none-any.whl
Uploading django-configurations-2.0.dev0.tar.gz
/home/travis/virtualenv/python3.5.0/lib/python3.5/site-packages/setuptools/dist.py:283: UserWarning: The version specified requires normalization, consider using '2.0.dev0' instead of '2.0.dev'.
  self.metadata.version,
running upload_docs
creating build/docs
error: no files found in upload directory 'build/docs'
```

**Comment by jezdez** [comment]
@codingjoe Just wrote you an email after discovering the [wrong 2.0dev release to PyPI](https://pypi.python.org/pypi/django-configurations/2.0.dev0) myself. This wasn't a failed auto deploy but a mistake on your part not bumping up the version nummer in the `__init__.py` file correctly.

Sorry all for the kerfuffle, I'll fix this asap.

**Comment by jezdez** [comment]
@codingjoe For the future, since you've discovered the issue with the deploy over a month ago, I want to stress that for such cases in which Jazzband infrastructure doesn't work as intended you need to open a [ticket for the Roadies](https://jazzband.co/roadies). Thanks!

**Comment by jezdez** [comment]
I just released 2.0 as the replacement for the broken 1.10 release. See the [changelog](http://django-configurations.readthedocs.io/en/stable/changes/#v2-0-2016-07-29) for the details what changed.

In case you're curious, it became 2.0 and not say 1.10.1 since the changes in it also contained a deprecation of main features (e.g. support for Python 2.6 and 3.2) and 1.10 basically should never have happened in the first place.

Thanks all for the patience, sorry again for the trouble.
